docs/aqua-book/language/types.md

```diff
@@ -16,15 +16,24 @@ Scalar types follow the Wasm IT notation.
 
 You can pass booleans (true, false), numbers, double-quoted strings as literals.
 
-## Products
+## Structures
 
 ```aqua
-data ProductName:
-  field_name: string
-
-data OtherProduct:
-  product: ProductName
-  flag: bool
+data InnerStruct:
+    arr: []string
+    num: u32
+
+data SomeStruct:
+    str: string
+    num: u64
+    inner: InnerStruct
+
+-- structures can be filled with elements in aqua code
```

Comment:
I'm not sure that `structures` consist of `elements`, and that you `fill` the structure when create an instance of it

Reply:
structures consist of fields, and what you do is `create`, `initialize`, `instantiate`.